### ISSUE TITLE
Show proposal comments in review screen

### ISSUE THREAD

**Issue by ceci** [agile-alliance-brazil/submissions#120]
**As a...** reviewer
**I want...** to be able to ask an author for clarifications regarding his submission, during the submission period
**So that...** the review process is more interactive and fair.

**Comment by hugocorbucci** [comment]
How does that work? Is it something that only reviewers have access to? Is it a form in the review page? Is it just comments again?

**Comment by ceci** [comment]
This is also kinda working well with the reviewers comments. Maybe, just maybe, this could turn into "showing session comments on the review process".

What do you think, @raphaelmolesim, @ericabriones and @gmotta?

**Comment by gmotta** [comment]
I'm a bit confused. The issue says "during submission period".

On the review pages (after submission period), as reviwers, do we see the comments? I believe not, but we can always read them or open on another tab.

On the submission page (during and after) I believe it is working just fine. Reviewers add comments and authors are replying.

**Comment by hugocorbucci** [comment]
How are we feeling about this now?

**Comment by ceci** [comment]
I feel like we've been successfully using the comments for that purpose ("allowing clarification"), and this issue should turn into making this information easily accessible at the time of the reviews.

**As a...** reviewer
**I want...** to see the comments a session had during the submission period
**So that...** I can take into consideration the author's engagement on enhancing the submissions, and also the suggestions and clarification requests received.

**Comment by ericabriones** [comment]
I agree with the new user story @Ceci proposed. I actually didn't even realized that they weren't visible during the review period, but I guess that's because I work with so many tabs open that I end up seeing them anyway. Nonetheless the rule should remain that only modifications applied to the submission will be considered.

**Comment by gmotta** [comment]
I was looking into how comments were implemented in the 1st place back in 2010.
https://github.com/agile-alliance-brazil/submissions/commit/176d3670b6f92c1779df2bb357d7361ea4d760ac#diff-2a0701eaa5d32b4adbeb9d9198cb07a0

I think this is a good enhancement, but again, we have been living without it. Reviewers just lookup comments (or ignore them). As for the people on this thread, it seems that everyone opens the review in a tab and the actual proposal on another to read the comments.
